### ISSUE TITLE
Consider removing carousels as their are not accessible

### ISSUE THREAD

**Issue by gagarine** [swiss/styleguide#132]
From bootstrap
"The carousel component is generally not compliant with accessibility standards. If you need to be compliant, please consider other options for presenting your content."

http://www.webaxe.org/carousels-and-aria-tabs/

http://shouldiuseacarousel.com/

http://www.creativebloq.com/accessibility-expert-warns-stop-using-carousels-7133778
